ci/run.bash

```diff
@@ -15,11 +15,12 @@ case "$(uname -s)" in
 esac
 
 case "$TARGET" in
-  # these platforms aren't supported by ring:
-  powerpc* ) ;;
+  # these platforms aren't supported by aws-lc-rs:
+  powerpc64* ) ;;
   mips* ) ;;
-  riscv* ) ;;
-  s390x* ) ;;
+  loongarch* ) ;;
+  *netbsd* ) ;;
+  *illumos* ) ;;
```

Comment:
So does aws-lc-rs work on non-64-bit PowerPC, RISC-V and S390x?

Reply:
@djc Yes, each addition/deletion has been manually tested on CI; see https://github.com/rust-lang/rustup/actions/runs/12384929706.